scripts/fuzz.sh

```diff
@@ -116,6 +120,14 @@ main()
     -DCONFIG_ASAN=y
   )
 
+  local debug_configs=(
```

Comment:
It's pretty amazing that this does NOT git conflict considering I removed the entire `fuzz_configs` variable above (and more) in merged commit 577c9ae3e75 and replaced it with `local conf_files_list=...` which looks entirely different! Yet neither `git cherry-pick` nor `git merge` seem to bother - wow!

Even if not a git conflict, this is still very much a logical conflict so this PR should not be merged as is right now.

There will likely be another conflict with incoming #9409, so maybe wait for that to be merged too.